src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -506,7 +517,17 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
   /** The rest of the collection without its `n` last elements. For
     *  linear, immutable collections this should avoid making a copy.
     */
-  def dropRight(n: Int): C = fromSpecificIterable(View.DropRight(coll, n))
+  def dropRight(n: Int): C = {
+    val b = newSpecificBuilder()
+    if (n >= 0) b.sizeHint(coll, -n)
```

Comment:
This version of `sizeHint` takes a collection and a delta to apply to the collection’s size. Here we want to say that the size of the result will be the size of `coll` minus the number of dropped elemnts. I could write explicitly `sizeHint(coll, delta = -n)`, for clarity.